packages/paste-core/components/radio-button-group/src/StyledRadioButtonGroupWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+import {styled, css} from '@twilio-paste/styling-library';
+
+export const AttachedRadioButtonGroupWrapper = styled.span(
+  css({
+    // NOTE: allow focus indication to remain visible on all sides of a button despite
+    // sibling elements being rendered immediately adjacent under the attached styles.
+    // Not applied to hover for consistent styling with borders.
+
+    '& > label:focus-within': {
```

Comment:
question (non-blocking): why can't we re-use/compose ButtonGroup in RadioButtonGroup to apply the styling - how is this different?

Reply:
Oh! good point, this work started before ButtonGroup was finished. It can totally just re-export RadioGroup.

Reply:
these are different enough that they should be separate (span vs label), so we'll leave as is.